body.tex

```diff
@@ -0,0 +1,51 @@
+
+
+\section{Introduction}
+\subsection{Scope}
+This document covers releases of software from the Data Management Subsystem of LSST.
+It discusses the delineation between the Data Facility as an operational entity and DM producing and testing software.
+
+\section{Release management}\label{sect:relman}
+
+The release manager will be the final owner of this process the current understanding is outlined  here.
+
+\subsection{Preparation of Releases}\label{sect:relprep}
+DM develops code in github following the developer guidelines \url{https://developer.lsst.io/}.
+This includes automated testing and continuous integration. Tested releases are tagged by SQuARE
+weekly and major releases are made each Cycle (six months).
```

Comment:
It would be useful to specify exactly what is being released every six months. We currently have many packages under github.com/lsst which do not get tagged for release. And those are just the simple cases: what about github.com/lsst-sqre? Firefly? etc...